core/src/player.rs

```diff
@@ -1883,7 +1889,7 @@ impl PlayerBuilder {
                 mouse_cursor_needs_check: false,
 
                 // Misc. state
-                rng: SmallRng::seed_from_u64(chrono::Utc::now().timestamp_millis() as u64),
+                rng: SmallRng::seed_from_u64(random_seed),
                 system: SystemProperties::default(),
```

Comment:
Why not simply use [`get_current_date_time()`](https://github.com/ruffle-rs/ruffle/blob/8efd69b5255ea1c0f0cdd2a87f01ff1c9077bf35/core/src/locale.rs#L8) here? It already checks for the `deterministic` feature, so it won't be needed here.

Reply:
That doesn't quite feel as explicit and could be subject to accidentally being refactored away.

Reply:
I'd argue that being implicit here isn't a bad thing, and even good. For instance, `get_current_date_time()` checks for `cfg!(test)`, while this currently doesn't. I believe that checking for `cfg!(test)` is desirable, but adding it here would introduce code duplication. In general, I think `chrono::Utc::now()` should never be used directly, as this would always introduce non-determinism.